src/components/CreateAccount.jsx

```diff
@@ -1,9 +1,285 @@
-import React from 'react'
+import React from "react";
+import { useState, useEffect } from "react";
+import { useForm } from "react-hook-form";
+import { Link, useNavigate } from "react-router-dom";
+import {Container,
+        TextField,
+        Box,
+        MenuItem,
+        Typography,
+        Grid,
+        FormControl,
+        InputLabel,
+        Input,
+        FormHelperText,
+        Button,
+        Select} from "@mui/material";
+
+import FileBase64 from 'react-file-base64';
+
+import MyButton from "../components/Button/MyButton";
+import { createAccountPageStyles } from "../CreateAccount.styles.js";
+
+import BrandLogo from "../assets/fiftyfive-logo.png";
+import CreateAccountImage from "../assets/create-account.jpg";
+import LoginPageImage from '../assets//loginPageImage1.gif';
 
 const CreateAccount = () => {
+  const { classes } = createAccountPageStyles();
+
+  const [profilePic, setProfilePic] = useState("");
+
+  const { register, handleSubmit, formState: { errors }, } = useForm();
+  const navigate = useNavigate();//to redirect user to login page after successfull signup
+
+  const axios = require("axios");
+
+  //method to create user account
+  const onSubmit = async (data) => {
+    data.profilePic = profilePic;
+    console.log(data);
+
+    let config = {
+      method: "post",
+      maxBodyLength: Infinity,
+      url: "http://192.168.1.34:8484/v1/signup",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      mode: "no-mode",
+      referrerPolicy: "no-referrer",
+      data: data,
+    };
+
+    await axios
+      .request(config)
+      .then((response) => {
```

Comment:
If you are using await then don't use "then" bind it in try catch

Reply:
Okay !